implement-shell-tools/README.md

```diff
@@ -0,0 +1,9 @@
+# Implement shell tools
+
+Your task is to re-implement shell tools you have used.
```

Comment:
General thought: do we need to spell it out that the scripts should be run via the `node`/`python` executable? In other words, they won't be making their scripts executable directly?

I would expect some confusion over this since technically they can't run `cat.js` like they can run `cat`

Reply:
Good shout - added a clarification